src/Utility/GetDeclaringSourceTrait.php

```diff
@@ -48,10 +48,7 @@ protected function getDeclaringSource(Node\Expr $expr): ?string
             return $declaringTrait->getName();
         }
         $declaringClass = $exprReflection->getDeclaringClass();
-        if ($declaringClass !== null) {
-            return $declaringClass->getName();
-        }
-        return null;
+        return $declaringClass->getName();
```

Comment:
Appeantly this can never be null.

Reply:
Yeah, PHPStan is smarter than PhpStorm given the surrounding context.